src/psyclone/psyir/nodes/reference.py

```diff
@@ -172,6 +172,10 @@ def reference_accesses(self, var_accesses):
 
         '''
         sig, all_indices = self.get_signature_and_indices()
+        if isinstance(self.symbol.interface, ImportInterface) and \
```

Comment:
I think you can just do `if self.symbol.is_import and ...` here? If so, you can get rid of the import of ImportInterface again.

Reply:
Oh, that's nice.